scripts/create-project.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+require __DIR__ . '/functions/autoload.php';
+
+use function Studiometa\WPInstaller\{getSalts, randomChars, randomNumber, updateFile, runCommands};
+
+$name = basename( dirname( __DIR__ ) );
+
+updateFile(
+	'package.json',
+	[
+		1 => sprintf( "  \"name\": \"%s\",", $name ),
+		2 => "  \"version\": \"0.0.0\",",
+	]
+);
+
+updateFile(
+	'.ddev/config.yaml',
+	[
+		0 => sprintf( 'name: %s', $name ),
+	]
+);
+
+updateFile(
+	'README.md',
+	[
+		0 => sprintf( "# %s", $name ),
+		7 => sprintf( 'git clone git@gitlab.com:studiometa/%s.git', $name ),
+	]
+);
+
+runCommands(
+	'Initialize Git repository',
```

Comment:
**issue:** I don't think it's the role of this package to initialize git. Maybe we would like to create a project without git (only to test a feature locally). Moreover, it really depends on user access on repository.

**suggestion:** Maybe this script should be run in a manually separated script based on arguments.

Reply:
> issue: I don't think it's the role of this package to initialize git. Maybe we would like to create a project without git (only to test a feature locally). Moreover, it really depends on user access on repository.

I think that initializing Git is not out of scope for this repository. We can still create test projects locally with Git, it should not have an impact on a test. 

I can remove adding the origin as it is the action that could change between projects. 

> suggestion: Maybe this script should be run in a manually separated script based on arguments.

We might as well just do a `git init` command 😉